dist/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="../dist/main.js"></script>
+  <title>Weather Broadcast</title>
+</head>
+
+<body>
+
+  <main>
```

Comment:
we have two index.html in the project, please get rid of one.

Reply:
I use the ./src/index.html as a template to generate ./dist/index.html. I used html-webpack-plugin for that :smile:

Reply:
is there really a need to run our html through webpack?